.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
```

Comment:
We are not using travis-pro, kindly remove this as it has no effect. Unless you have a reason for it being here, if yes let me know.